### PR TITLE
only build css files for production dist creation

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

**Comment by erwinmombay** [comment]
@cramforce PTAL

**Comment by erwinmombay** [comment]
shave 25-30s for travis builds and dist creation

File: .travis.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ before_script:
 script:
   - npm run ava
   - gulp lint
-  - gulp build --fortesting
+  - gulp build --fortesting --css-only
```
Comment (cramforce) [inline .travis.yml]: Consider documenting the flag in `gulp help`

**Comment by cramforce** [comment]
LGTM

File: .travis.yml (hunk 1 above)
Comment (erwinmombay) [inline .travis.yml]: done.

**Pull request merged**